### DISCUSSION TITLE
Make Toolbar's Background Transparent Again

### DISCUSSION THREAD

**Discussion by gjaekel** [igniterealtime/openfire-pade-plugin#339] [Ideas]
With the design changes of the  latest Jitsi WebClient UI, the Toolbar area get a black, solid background. This "overlaps" a big area of the bottom row of the participants windows in tiled view.

![image](https://user-images.githubusercontent.com/19855721/117683880-c6a71380-b1b4-11eb-971a-a75aef9ede98.png)

As requested by one of my co-workers, I suggest a CSS patch to make the background transparent, again.

![image](https://user-images.githubusercontent.com/19855721/117684569-6b295580-b1b5-11eb-94f1-69e5441f8e57.png)

To archive this, following might be added to `custom_ofmeet.css`
```
/*<<< 20210510/gj<km */
.invite-more-content {
  background: rgba(0,0,0,.3);
}

.toolbox-content-items {
  background: rgba(0,0,0,.3);
}
/*>>>*/

```

@deleolajide @cool0707 @ClaudeStabile What's your opinion on this, please?

**Comment by ClaudeStabile** [comment]
@gjaekel Guido, I agree, currently a too large black box is coming over participants, That's a progress i agree with your suggestion and it comes from a user experience so.. :)
